_posts/2024-07-02-schematic-review-checklist.md

```diff
@@ -1,45 +1,45 @@
 ---
```

Comment:
Last couple things (making a comment to track them):
- update date to 7/17
- run prettier `npx prettier --prose-wrap always --write [post]` -- I've been running this manually lately but Noah has a PR up to add to it to the precommit hook. I would put this formatting in a separate commit since it will make changes hard to see.